main.go

```diff
@@ -39,6 +38,13 @@ var (
 	errParseECPoint  = "failed to parse EC point"
 )
 
+const (
+	// January 1, 2022
+	firstEpochTimestampMs int64 = 1640995200000
+	// One week
+	epochLengthTimeMs int64 = 604800000
```

Comment:
Can you construct this in place as `7*24*3600*1000` to make it easier to adjust in the future?

Reply:
sure